.github/workflows/app_deploy_develop.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish on Netlify
+on:
+  push:
+    branches:
+      - "develop"
```

Comment:
Nice! But hey, how comes it was working before? I'm confused 👀

Reply:
I disabled the Netlify automatic deployment so we only build and deploy the things we want to reduce costs with Netlify

Reply:
oh ok ok. Nice!